scripts/subscribeEventProof.js

```diff
@@ -117,26 +99,70 @@ async function main (networkName, bridgeContractAddress) {
     logger.info(`CENNZnet bridge deployed to: ${bridge.address}`);
     logger.info(`Executor: ${txExecutor.address}`);
 
-    let lastValidatorsSet = getValidatorAddedToBridge(bridge);
+    // For any kind of restart, we check if the last event proof generated on CENNZnet side, has been update on Eth side
+    logger.info('Check the last event proof generated on CENNZnet side, has been update on Eth side');
+    const lastEventProofIdFromCennznet = await api.query.ethBridge.notarySetProofId();
+    logger.info(`lastEventProofIdFromCennznet: ${lastEventProofIdFromCennznet.toString()}`);
+    const eventExistsOnEth = await bridge.eventIds(lastEventProofIdFromCennznet.toString());
+    logger.info(`eventExists on Ethereum: ${eventExistsOnEth}`);
+    try {
```

Comment:
it's good practice to scope the `try` block to only the part that requires the catch